docs/source/actions.rst

```diff
@@ -196,13 +196,19 @@ List of recipients of the message. It could be a channel or an user.
 
 SPIDERMON_SLACK_SENDER_NAME
 ---------------------------
+Name of app/bot/user created for serving notifications on desired channels.
 
 .. _SPIDERMON_SLACK_SENDER_TOKEN:
 
 SPIDERMON_SLACK_SENDER_TOKEN
 ----------------------------
 
-Your `Slack token`_.
+Read `here`_ to know more about Slack Tokens.
+Get your Slack Credentials by `creating an app <https://api.slack.com/apps>`_  here.
+
+The `SPIDERMON_SLACK_SENDER_TOKEN` is the `API Token` available in your Slack application dashboard. You can navigate to Browse apps> Custom Integrations > Bots > Edit Configuration. 
```

Comment:
Instead of describing this here, maybe we should find some upstream documentation that describes how to access your Slack token, and link ‘Slack token’ there instead of linking to the API documentation as we currently do.